### PR TITLE
Standardize download sections

### LINKED ISSUE #1727: Standardize download sections
### Discussed in https://github.com/privacyguides/privacyguides.org/discussions/1726

<div type='discussions-op-text'>

<sup>Originally posted by **Redre1l** September  7, 2022</sup>
So, in the password managers section Bitwarden has listed browser extension stores, Android, iOS stores etc.
While 1Password is in the same section and has no listed apps.

Also I don't think it makes sense to not list downloads in the mail and VPN sections.</div>

### PR DESCRIPTION
**Pull request by KaiTebay**
# Resolves #1727

## Standardized Order

|Download Type|Format|Details|
|---|---|---|
|Google Play Store|`[:simple-googleplay: Google Play]()`||
|App Store|`[:simple-appstore: App Store]()`|iOS, macOS, and iPadOS|
|GitHub, etc.|`[:simple-ICON: NAME]()`|Provided that they have a "Releases" section—with downloads—like GitLab and Codeberg|
|Windows|`[:simple-windows11: Windows]()`|.exe files|
|macOS|`[:simple-apple: macOS]()`|.dmg or .pkg files|
|Linux|`[:simple-linux: Linux]()`|.deb, .rpm, etc. *or* distribution-specific instructions|
|PWAs or Extensions|`[:octicons-globe-16: PWA]()` `[:simple-ICON: NAME]()`|Use the browser name. Order: Firefox, Chrome, Edge, Safari|
|Web|`[:octicons-browser-16: Web]()`|e.g. [Snowflake](https://www.privacyguides.org/tor/#snowflake)|
|FreeBSD|`[:simple-freebsd: FreeBSD]()`||
|OpenBSD|`[:simple-openbsd: OpenBSD]()`||
|NetBSD|`[:simple-netbsd: NetBSD]()`||

#### Official Website

- [Cryptomator](https://cryptomator.org/android) (Edit: auto-updates).
- [Signal](https://signal.org/android/apk/) (auto-updates).
- [Tor](https://www.torproject.org/download/#android) (unsure). :warning: For Review

#### GitHub+, added:

- [BitWarden](https://github.com/bitwarden/mobile/releases)
- [Brave](https://github.com/brave/brave-browser/releases)
- [Element](https://github.com/vector-im/element-android/releases)
- [FairEmail](https://github.com/M66B/FairEmail/releases)
- [Joplin](https://github.com/laurent22/joplin-android/releases)
- [Nextcloud](https://github.com/nextcloud/android/releases)
- [RethinkDNS](https://github.com/celzero/rethink-app/releases)
- [Session](https://github.com/oxen-io/session-android/releases)

### Flathub :warning: For Review

`Live?` Is the Flathub download currently listed on privacyguides.org?

|Name|Live?|Official?|Details|
|---|---|---|---|
|[1Password](https://support.1password.com/install-linux/#flatpak)|:red_circle:|:green_circle:|[Official Flatpak image](https://support.1password.com/install-linux/#flatpak), direct from 1password.com (not on Flathub)|
|[Akregator](https://flathub.org/apps/details/org.kde.akregator)|:green_circle:|:question:|Linked on homepage|
|[BitWarden](https://flathub.org/apps/details/com.bitwarden.desktop)|:green_circle:|:question:|Not linked on homepage|
|[Brave](https://flathub.org/apps/details/com.brave.Browser)|:red_circle:|:orange_circle:|[Linked](https://brave.com/linux/#release-channel-installation) as an unofficial image|
|[Briar](https://flathub.org/apps/details/org.briarproject.Briar)|:green_circle:|:question:|Not linked on homepage|
|[Cryptomator](https://flathub.org/apps/details/org.cryptomator.Cryptomator)|:green_circle:|:question:|Linked on homepage|
|[Element](https://flathub.org/apps/details/im.riot.Riot)|:red_circle:|:question:|Linked on homepage|
|[Evolution](https://wiki.gnome.org/Apps/Evolution/Flatpak)|:green_circle:|:question:|Linked on homepage ([support article](https://wiki.gnome.org/Apps/Evolution/Flatpak))|
|[Firefox](https://flathub.org/apps/details/org.mozilla.firefox)|:green_circle:|:question:|Linked in [support article](https://support.mozilla.org/en-US/kb/install-firefox-linux#w_install-from-flatpak)|
|[FreeTube](https://flathub.org/apps/details/io.freetubeapp.FreeTube)|:green_circle:|:question:|Same primary contributor ([FreeTube](https://github.com/FreeTubeApp/FreeTube/graphs/contributors) vs. [Flathub](https://github.com/flathub/io.freetubeapp.FreeTube/graphs/contributors/))
|[GNOME Feeds](https://flathub.org/apps/details/org.gabmus.gfeeds)|:green_circle:|:question:|Linked on homepage|
|[Joplin](https://flathub.org/apps/details/net.cozic.joplin_desktop)|:red_circle:|:orange_circle:|[Linked](https://discourse.joplinapp.org/t/unofficial-alternative-joplin-distributions/23703) as an unofficial image|
|[KeePassXC](https://flathub.org/apps/details/org.keepassxc.KeePassXC)|:green_circle:|:green_circle:|[Official](https://keepassxc.org/download/#linux) download|
|[LBRY](https://flathub.org/apps/details/io.lbry.lbry-app)|:red_circle:|:orange_circle:|[Listed](https://github.com/lbryio/lbry-desktop#install) as unofficial image|
|[LibreOffice](https://flathub.org/apps/details/org.libreoffice.LibreOffice)|:green_circle:|:question:|Linked on homepage|
|[Nextcloud](https://flathub.org/apps/details/com.nextcloud.desktopclient.nextcloud)|:red_circle:|:question:|Not linked on homepage|
|[OnlyOffice](https://flathub.org/apps/details/org.onlyoffice.desktopeditors)|:green_circle:|:green_circle:|[Official](https://www.onlyoffice.com/download-desktop.aspx) download|
|[OnionShare](https://flathub.org/apps/details/org.onionshare.OnionShare)|:red_circle:|:green_circle:|Recommended [download](https://docs.onionshare.org/2.5/en/install.html#install-in-linux)|
|[Session](https://flathub.org/apps/details/network.loki.Session)|:red_circle:|:orange_circle:|[GitHub issue](https://github.com/oxen-io/session-desktop/issues/2146)|
|[Signal](https://flathub.org/apps/details/org.signal.Signal)|:green_circle:|:question:|Not linked on homepage|
|[Standard Notes](https://flathub.org/apps/details/org.standardnotes.standardnotes)|:orange_circle:|:question:|Not linked on homepage|
|[Syncthing GTK](https://flathub.org/apps/details/me.kozec.syncthingtk) and [SyncThingy](https://flathub.org/apps/details/com.github.zocker_160.SyncThingy)|:red_circle:|:orange_circle:|Not linked on homepage|
|[Thunderbird](https://flathub.org/apps/details/org.mozilla.Thunderbird)|:green_circle:|:question:|Not linked on homepage|
|[Tor Browser Launcher](https://flathub.org/apps/details/com.github.micahflee.torbrowser-launcher)|:red_circle:|:question:|[GitHub](https://github.com/micahflee/torbrowser-launcher) not recently updated? [Security Design](https://github.com/micahflee/torbrowser-launcher/blob/develop/security_design.md)|
|[Tutanota](https://flathub.org/apps/details/com.tutanota.Tutanota)|:green_circle:|:question:|Not linked on homepage|

### Docker Hub :warning: For Review
- Included Docker Hub links for tools which already had them. There might be more tools where it makes sense to link to Docker Hub.
    - [EteSync](https://hub.docker.com/r/victorrds/etesync)
    - [Psono](https://hub.docker.com/r/psono/psono-client)

- [x] Please check this box to confirm you have disclosed any relevant conflicts of interest in your post.
- [x] Please check this box to confirm your agreement to publish your work under the [Creative Commons Attribution-NoDerivatives 4.0 International](https://github.com/privacyguides/privacyguides.org/blob/main/LICENSE) license, and to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform and distribute your contribution as part of our project.

### CODE REVIEW

**Comment by KaiTebay** [comment]
**Points of Note**
- Included Cryptomator [Android](https://cryptomator.org/android) (I don't know whether it auto-updates).
- Included Signal [Android](https://signal.org/android/apk/) (because it auto-updates).
- Included Tor [Android](https://www.torproject.org/download/#android) (unsure).

- Included Docker Hub links for tools which already had them. There might be more tools where it makes sense to link to Docker Hub.
    - [EteSync](https://hub.docker.com/r/victorrds/etesync)
    - [Psono](https://hub.docker.com/r/psono/psono-client)

File: docs/productivity.en.md [hunk 1]
```diff
@@ -26,15 +26,15 @@ For other platforms, consider below:
 
     ??? downloads
 
+        - [:simple-googleplay: Google Play](https://www.libreoffice.org/download/android-and-ios/)
+        - [:simple-appstore: App Store](https://www.libreoffice.org/download/android-and-ios/)
         - [:simple-windows11: Windows](https://www.libreoffice.org/download/download/)
         - [:simple-apple: macOS](https://www.libreoffice.org/download/download/)
         - [:simple-linux: Linux](https://www.libreoffice.org/download/download/)
         - [:simple-flathub: Flatpak](https://www.libreoffice.org/download/download/)
```
Comment (jonaharagon) [inline docs/productivity.en.md:34]:
```suggestion
        - [:simple-flathub: Flathub](https://flathub.org/apps/details/org.libreoffice.LibreOffice)
```

File: docs/real-time-communication.en.md [hunk 2]
```diff
@@ -122,14 +123,13 @@ Session has a [whitepaper](https://arxiv.org/pdf/2002.04609.pdf) describing the
 
     ??? downloads
 
-        - [:simple-flathub: Flatpak](https://flathub.org/apps/details/org.briarproject.Briar)
         - [:simple-googleplay: Google Play](https://play.google.com/store/apps/details?id=org.briarproject.briar.android)
-        - [:simple-fdroid: F-Droid](https://f-droid.org/packages/org.briarproject.briar.android)
+        - [:simple-flathub: Flatpak](https://flathub.org/apps/details/org.briarproject.Briar)
```
Comment (jonaharagon) [inline docs/real-time-communication.en.md:127]:
```suggestion
        - [:simple-flathub: Flathub](https://flathub.org/apps/details/org.briarproject.Briar)
```

File: docs/calendar-contacts.en.md [hunk 3]
```diff
@@ -23,14 +23,13 @@ Calendars and contacts contain some of your most sensitive data; use products th
 
     ??? downloads
 
-        - [:octicons-browser-16: Web](https://mail.tutanota.com/)
+        - [:simple-googleplay: Google Play](https://play.google.com/store/apps/details?id=de.tutao.tutanota)
+        - [:simple-appstore: App Store](https://apps.apple.com/us/app/tutanota/id922429609)
         - [:simple-windows11: Windows](https://tutanota.com/blog/posts/desktop-clients/)
         - [:simple-apple: macOS](https://tutanota.com/blog/posts/desktop-clients/)
         - [:simple-linux: Linux](https://tutanota.com/blog/posts/desktop-clients/)
         - [:simple-flathub: Flatpak](https://flathub.org/apps/details/com.tutanota.Tutanota)
```
Comment (jonaharagon) [inline docs/calendar-contacts.en.md:31]:
```suggestion
        - [:simple-flathub: Flathub](https://flathub.org/apps/details/com.tutanota.Tutanota)
```

File: docs/encryption.en.md [hunk 4]
```diff
@@ -24,13 +24,13 @@ The options listed here are multi-platform and great for creating encrypted back
 
     ??? downloads
 
+        - [:simple-googleplay: Google Play](https://play.google.com/store/apps/details?id=org.cryptomator)
+        - [:simple-appstore: App Store](https://apps.apple.com/us/app/cryptomator-2/id1560822163)
+        - [:simple-android: Android](https://cryptomator.org/android)
         - [:simple-windows11: Windows](https://cryptomator.org/downloads)
         - [:simple-apple: macOS](https://cryptomator.org/downloads)
         - [:simple-linux: Linux](https://cryptomator.org/downloads)
         - [:simple-flathub: Flatpak](https://flathub.org/apps/details/org.cryptomator.Cryptomator)
```
Comment (jonaharagon) [inline docs/encryption.en.md:33]:
```suggestion
        - [:simple-flathub: Flathub](https://flathub.org/apps/details/org.cryptomator.Cryptomator)
```

File: docs/passwords.en.md [hunk 5]
```diff
@@ -25,13 +25,12 @@ These password managers sync your passwords to a cloud server for easy accessibi
 
     ??? downloads
 
+        - [:simple-googleplay: Google Play](https://play.google.com/store/apps/details?id=com.x8bit.bitwarden)
+        - [:simple-appstore: App Store](https://apps.apple.com/app/bitwarden-password-manager/id1137397744)
+        - [:simple-github: GitHubx](https://github.com/bitwarden/mobile/releases)
         - [:simple-windows11: Windows](https://bitwarden.com/download)
-        - [:simple-apple: Mac App Store](https://apps.apple.com/app/bitwarden/id1352778147)
         - [:simple-linux: Linux](https://bitwarden.com/download)
         - [:simple-flathub: Flatpak](https://flathub.org/apps/details/com.bitwarden.desktop)
```
Comment (jonaharagon) [inline docs/passwords.en.md:33]:
```suggestion
        - [:simple-flathub: Flathub](https://flathub.org/apps/details/com.bitwarden.desktop)
```

**Review by jonaharagon — CHANGES_REQUESTED** [review]
I think we should treat Flathub as another app store, so Flatpak links should be renamed to Flathub links, and any Flatpaks that aren't on Flathub should be labeled as a generic "Linux" download instead of Flatpak, if that makes sense.

**Comment by KaiTebay** [comment]
> Another note is that since we're keeping Flathub links, we should comb through all of the tools we recommend and add missing flatpaks for them too. KeePassXC comes to mind. Also, should we only list official flatpaks, or are community maintained ones valid as well?

I'll update all of the tools with Flathub links and mark whether they're maintained officially or by the community so that we can decide.

**Review by dngray — APPROVED** [review]

**Pull request merged**